pkg_building.Rmd

```diff
@@ -197,8 +204,9 @@ Please do not list editors as contributors. Your participation in and contributi
 
 ## Examples
 
-* Include extensive examples in the documentation. In addition to demonstrating how to use the package, these can act as an easy way to test package functionality before there are proper tests. However, keep in mind we require tests in contributed packages. If you prefer not to clutter up code with extensive documentation, place further documentation/examples in files in a `man-roxygen` folder in the root of your package, and those will be combined into the manual file by the use of `@template <file name>`, for example. You can run examples with `devtools::run_examples()`.
+* Include extensive examples in the documentation. In addition to demonstrating how to use the package, these can act as an easy way to test package functionality before there are proper tests. However, keep in mind we require tests in contributed packages. 
 
+*  You can run examples with `devtools::run_examples()`.
```

Comment:
@maelle should we add that we want pkg maintainers to make sure they run examples wrapped in `\dontrun{}` or `\donttest{}` somewhere?  most likely on CI, either travis or appveyor or both. `r_check_args: "--as-cran --run-dontrun"`

Reply:
@maelle what did you think about this comment?

Reply:
good point!

Reply:
can you add this point? Thanks!

Reply:
sure